common/algorithms/hashes.h

```diff
@@ -6,17 +6,20 @@
 #define ENGINE_HASHES_H
 
 #include <array>
+#include <cstddef>
 #include <cstdint>
 #include <functional>
 #include <iterator>
-#include <numeric>
 #include <utility>
 
 #include "common/type_traits/range_value_type.h"
 #include "common/wrappers/span.h"
 
 namespace vk {
 
+template<typename T>
+T murmur_hash(const void *ptr, size_t len, size_t seed = static_cast<size_t>(0xc70f6907UL)) noexcept;
```

Comment:
As I saw in code there's a pattern
 ```C++
 std::string_view name{...};
 vk::murmur_hash(name.data(), name.size());
 ```
Maybe do signature murmur_hash like  `T murmur_hash(std::string_view`,...)?

Reply:
I don't want to require `std::string_view` in order to use the hashing algorithm